pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "celi-framework"
-version = "0.0.9"
+version = "0.0.11"
```

Comment:
See above, this gets changed during release.  Keep it at 0.0.0

It doesn't hurt to change it, but it's unnecessary and leaving it at 0.0.0 makes that clearer.  Probably we should add a comment explaining that.